test/component/login_test.dart

```diff
@@ -162,48 +178,5 @@ void main() {
       final loginPage = find.byType(LoginPage);
       expect(loginPage, findsOneWidget);
     });
-
-    testWidgets("Login to Create Account Page to Home Page flow",
```

Comment:
By humble curiosity, why was this test deleted?

Reply:
It's because it is already contained in the test "Login flow to HomePage and log out" (and in the "Widgets display" group for the slogan and button testing).

Reply:
Indeed, it was redundant with other tests. Well seen!